docs/source/index.rst

```diff
@@ -4,36 +4,36 @@
    contain the root `toctree` directive.
 
 
-.. image:: ../logos/lightly_logo_crop.png
-  :width: 600
-  :alt: Lightly
+.. image:: ../logos/lightly_SSL_logo_crop.png
+   :width: 600
+   :align: center
+   :alt: Lightly
 
 
 Documentation
 ===================================
 
 .. note:: These pages document the Lightly self-supervised learning library.
-          If you are looking for Lightly Worker Solution to easily process millions
-          of samples and run powerful active learning algorithms on your data
-          please follow
+          If you are looking for the Lightly Worker Solution to easily process millions
+          of samples and run powerful `active learning algorithms <https://docs.lightly.ai/docs/selection>`_
+          on your data, please follow the
           `Lightly Worker documentation <https://docs.lightly.ai/>`_.
 
-Lightly is a computer vision framework for self-supervised learning.
+Lightly SSL is a computer vision framework for self-supervised learning.
 
-With Lightly you can train deep learning models using self-supervision. 
+With Lightly SSL you can train deep learning models using self-supervision. 
 This means, that you don’t require any labels to train a model. 
-Lightly has been built to help you understand and work with large unlabeled 
+Lightly SSL has been built to help you understand and work with large unlabeled 
 datasets. It is built on top of PyTorch and therefore fully compatible with 
 other frameworks such as Fast.ai.
 
 
-Lightly
--------
+Lightly AI
+----------
 
 - `Homepage <https://www.lightly.ai>`_
-- `Web-App <https://app.lightly.ai>`_
-- `Documentation <https://docs.lightly.ai/self-supervised-learning/>`_
-- `Lightly Solution Documentation (Lightly Worker & API) <https://docs.lightly.ai/>`_
+- `Lightly Worker Solution Documentation (Lightly Worker & API) <https://docs.lightly.ai/>`_
```

Comment:
Minor:
```suggestion
- `Lightly Worker Documentation <https://docs.lightly.ai/>`_
```

Reply:
During the original refactoring it was called Lightly Worker Solution. Sometimes its also Lightly Solution. So I will keep it (but remove the part in parenthesis)